NightscoutServiceKitUI/Views/ServiceStatusView.swift

```diff
@@ -39,6 +41,14 @@ struct ServiceStatusView: View, HorizontalSizeClassOverride {
                     Text(String(describing: viewModel.status))
                 }
                 .padding()
```

Comment:
I'm not a SwiftUI expert yet - I haven't figured out how to remove the blue text and get a chevron in place. Any suggestions welcome but not sure also if this is a blocker yet.

![IMG_CCA45DF4316D-1](https://user-images.githubusercontent.com/3207996/132065354-e9fc575b-8bbc-4d73-a674-554f33cdf00c.jpeg)